test/ejabberd_tests/tests/rest_client_SUITE.erl

```diff
@@ -87,6 +88,24 @@ msg_is_sent_and_delivered_over_sse(ConfigIn) ->
 
     stop_sse(Conn).
 
+msg_is_sent_and_delivered_over_sse_muc(Config) ->
```

Comment:
How about messages with notifications about users joining/leaving the room? I think a testcase for that would be useful.